### ISSUE TITLE
XHR failed when fetching extensions.

### ISSUE THREAD

**Issue by josephplpriest** [microsoft/vscode#222125]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
1.91.1
f1e16e1e6214d7c44d078b1f0607b2388f29d729
x64

- OS Version: 
Description:	Ubuntu 24.04 LTS
Release:	24.04
Codename:	noble

Steps to Reproduce:
1. Open VS Code
2. Select extensions
3. When fetching popular extensions, get "Error while fetching extensions. XHR failed"

Netlog output:

[netlog.json](https://github.com/user-attachments/files/16290157/netlog.json)

**Comment by chwkrvn** [comment]
I am experiencing the same on macOS 14.5, VSCode 1.91.1 (f1e16e1e6214d7c44d078b1f0607b2388f29d729). Occurs even when overwriting `--user-data-dir` and `--extensions-dir` to empty directories

**Comment by jordanss1** [comment]
I am getting the same. I am switching to Linux and just did a fresh install and none of my Settings/Extensions are being restored.

Is [Visual Studio Marketplace](https://marketplace.visualstudio.com/) down for anyone else?

**Comment by jcmdln** [comment]
It appears that the vscode marketplace is down:
![image](https://github.com/user-attachments/assets/4223cacf-966a-4839-a34f-4b925017a973)

There is currently an Azure outage which may be related, though I am just guessing: https://azure.status.microsoft/en-us/status

**Comment by Aisuko** [comment]
Both version of VSCode on macOS and Ubuntu all didn't work.
![Screenshot 2024-07-19 at 10 33 51 AM](https://github.com/user-attachments/assets/a8beb503-b200-4f5e-aa75-dddfffcd98eb)

**Comment by pitikorn32** [comment]
same issue

**Comment by jamesongithub** [comment]
thought this was client side! hopefully they can resolve it soon

**Comment by seunghyun0522** [comment]
![image](https://github.com/user-attachments/assets/23dc51c6-534e-48c6-8eba-f28cf78cb97e)

same issue

**Comment by vanessa-propeller** [comment]
SamedTTT
![2024-07-19 09 14 16](https://github.com/user-attachments/assets/62b9c25e-49f9-4393-83c7-191749424f34)

**Comment by someburner** [comment]
How about making extensions available to download from other sources...

**Comment by jtguibas** [comment]
Please fix!

**Comment by MerielVaren** [comment]
so it aint just appear on my laptop? god i was trying to figure it out the whole morning, plz fix it

**Comment by YUL-git** [comment]
Me too... I delete all proxy setting.json 😂

**Comment by nixgnauhcuy** [comment]
same issue! XHR failed

**Comment by myjconan** [comment]
same pl

**Comment by Lagee666** [comment]
same issue :((

**Comment by Ebbjik** [comment]
Oh my God, I've been tinkering all day, and it's actually a server-side problem?

**Comment by lieeh** [comment]
i have tried deleting setting.json, resetting my connection setting, reinstall and many orther things that i did. and it's actually just server down?  😢

**Comment by Gu7z** [comment]
All we can do is wait 😅

**Comment by mk965** [comment]
Unbelievable error, why is there such a large-scale paralysis

**Comment by nyanky0** [comment]
same, then its client issue then :(

**Comment by AhmedSetta2022** [comment]
damn i was trying to solve it for over 3 hrs
so its server problem not only me

**Comment by phamchin20** [comment]
me too ;-;
![image](https://github.com/user-attachments/assets/cc18dd17-4ea5-436d-b3ce-2ed3d6395232)

**Comment by greemle** [comment]
was just looking into this lmao, read one too many stack overflow articles

**Comment by hjplumtree** [comment]
Thanks for creating the thread!

**Comment by AhmedSetta2022** [comment]
shouldnt we send aticket ? or not possible ?

**Comment by wzastri** [comment]
same issue with the latest VScode version on macOS Sonoma 14.5.
tried many operations on proxy on internet and still not work...

**Comment by Gu7z** [comment]
> shouldnt we send aticket ? or not possible ?

I really think that azure should know for know that they services are off 😹

**Comment by g1anito** [comment]
Thank you for the free headache, please keep up these promotions.

**Comment by Aiden-Liu2008** [comment]
MacOS Sonoma 14.2 is also not working, I have tried to turn off the proxy strict SSL and change the proxy support,  it must be the problem of the server. Thanks for creating the thread!

**Comment by bcSatoshi** [comment]
same

**Comment by SOOBINIM** [comment]
same

**Comment by dantedevenir** [comment]
I was close to formatting the computer believing that they had sneaked in a MITM

**Comment by zyp1573** [comment]
![image](https://github.com/user-attachments/assets/b2e4eb62-b399-4405-a2f0-b0fa6ef2316f)

**Comment by JEricDevZ** [comment]
The same issue.
Windows 10 Enterprise 22H2 19045.4529. Tried both with Proxy or No Proxy.

**Comment by joeltheai** [comment]
i almost reset my whole laptop thinking my pc was pwned and some proxy stuff was messed up. spend my whole night on this :/

**Comment by friendshipity** [comment]
I'm not alone 😊

**Comment by liuzhoou** [comment]
<img width="349" alt="image" src="https://github.com/user-attachments/assets/92b15ac0-46b6-48aa-99e9-f1efb30442d5">
🤖

**Comment by Achen311** [comment]
i'm cofused all morning

**Comment by jimlin-mfi** [comment]
Same here!
<img width="332" alt="image" src="https://github.com/user-attachments/assets/c374a940-58c8-481d-bea4-c8f5a7435b3c">

**Comment by triple-Mu** [comment]
WTF? The earth is exploding!

**Comment by Ser4ph4** [comment]
I was almost setting my computer on fire, thinking the problem was on my side: kkkkkkkkkkkkkkkk

**Comment by NakagawaKouki** [comment]
OMG...I've initialized vscode...It was a server problem.

**Comment by eastera** [comment]
Same issue here.
Windows 10 Enterprise 22H2 19045.4529. Tried both either with a proxy or without a proxy.

**Comment by redj-4** [comment]
Bro after I reset my mac today, I thought something went wrong but its a server side problem

**Comment by MerielVaren** [comment]
The worst thing is that when i reinstall vscode already it still didnt work, then hopelessly i opened github and saw its a server problem😅

**Comment by JakeDeisley** [comment]
Well, glad I can just wait. Was looking through stack overflow articles, and trying some things without luck.

Weeeeeeee

**Comment by stiv2202** [comment]
this is so annoying! I thought it was my fault. 😮‍💨

**Comment by stiv2202** [comment]
btw could somebody help me to recover my github account? :/ it was hacked some weeks ago and I could fix it, but it seems Github still has it flagered as Spam 😢

**Comment by ganghyunnnn** [comment]
It's working for me now. Let's go guys!

**Comment by tranduc01** [comment]
Everything seems back to normal

**Comment by Ser4ph4** [comment]
**again work kkkkkkkk goo code**

**Comment by Ebbjik** [comment]
yes, it looks like he's back to normal, let's get on with the code

**Comment by Lagee666** [comment]
It's ok now. Time to work.

**Comment by jimlin-mfi** [comment]
Yeah, it's working now!

**Comment by huanghui1998hhh** [comment]
Still happens
